src/parser/generated_parser.ts

```diff
@@ -29,7 +29,7 @@ export class GeneratedParser extends Parser {
         //# file: statements? $
         let a, endmarker, mark;
         mark = this.mark();
-        if ((a = this.statements() || 1) && (endmarker = this.expect("ENDMARKER"))) {
+        if (((a = this.statements()), 1) && (endmarker = this.expect("ENDMARKER"))) {
```

Comment:
this seems to work a is assigned to this.statements and the statement is always truthy

Reply:
Ha ok :D